web/scripts/rate_all.py

```diff
@@ -100,8 +99,10 @@ def rate_all(t_from=None, t_to=None, iters=200, lam=.22):
             for k,v in rating_prior.items():
```

Comment:
Also, this seems like the kind of renormalization that should happen after the algorithm is finished, but I can also see why it would help stabilize the iterations within the loop.